src/main/java/org/folio/rest/impl/InstanceStorageBatchAPI.java

```diff
@@ -31,7 +31,7 @@
 
   private static final String INSTANCE_TABLE = "instance";
   private static final String BATCH_SIZE_KEY = "inventory.storage.max.batch.size";
-  private static final int MAX_BATCH_SIZE = Integer.parseInt(MODULE_SPECIFIC_ARGS.getOrDefault(BATCH_SIZE_KEY, "50"));
+  private static final int MAX_BATCH_SIZE = Integer.parseInt(MODULE_SPECIFIC_ARGS.getOrDefault(BATCH_SIZE_KEY, "4"));
```

Comment:
Where do `MODULE_SPECIFIC_ARGS` come from?

Reply:
It's an argument that can be specified in command that runs jar file for module on deployment. I will update the README with an example.